installation/upgrade-notes.md

```diff
@@ -97,15 +126,20 @@ the final record will be composed as follows:
 
 ## Fluent Bit v1.1
 
-If you are upgrading from **Fluent Bit &lt;= 1.0.x** you should take in consideration the following relevant changes when switching to **Fluent Bit v1.1** series:
+If you are upgrading from Fluent Bit &lt;= 1.0.x review the following relevant
```

Comment:
```suggestion
If you are upgrading from Fluent Bit 1.0.x, review the following relevant
```

Reply:
Seems like garbage characters.

Reply:
oh no it's for  <1.0.x in the UI - versions before 1.0.  Which means the text is still wrong stylistically but it's not incorrect in the display